src/WarcraftLegacies.Source/Setup/Legends/LegendScarlet.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using MacroTools;
+using MacroTools.ArtifactSystem;
+using MacroTools.Extensions;
+using MacroTools.LegendSystem;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+#pragma warning disable CS1591
+
+namespace WarcraftLegacies.Source.Setup.Legends
+{
+  /// <summary>
+  /// Responsible for creating and storing all Lordaeron <see cref="Legend"/>s.
+  /// </summary>
+  public sealed class LegendScarlet
+  {
+    public LegendaryHero Saiden { get; }
+    public LegendaryHero Renault { get; }
+    public LegendaryHero Brigitte { get; }
+    public LegendaryHero Sally { get; }
+
+    /// <summary>
+    /// Sets up all Lordaeron <see cref="Legend"/>s.
+    /// </summary>
+    public LegendScarlet()
+    {
+
+      Saiden = new LegendaryHero("Crusader Lord Saiden Dethrotan")
```

Comment:
Are you sure you want the title and names for all of these heroes? This name should typically be whatever they actually get referred to as. So "Saiden Dethrotan" here is probably better.